app/models/dossier.rb

```diff
@@ -766,8 +763,8 @@ def restore_dossier_and_destroy_deleted_dossier(author)
     log_dossier_operation(author, :restaurer, self)
   end
 
-  def discard_and_keep_track!(author, reason)
-    if termine? && author_is_administration(author)
+  def delete_and_keep_track!(author, reason)
```

Comment:
```suggestion
  def hide_and_keep_track!(author, reason)
```

J'ai l'impression que ca correspond plus à ce qu'on fait

Reply:
La méthode set un des flag `hidden_by_*` et dans certains cas ajoute un log - je trouve le nom raisonnable. Quelle serai ta proposition ?

Reply:
`hide_and_keep_track!`